www/src/pages/editor/EditorPage.tsx

```diff
@@ -20,12 +21,19 @@ export const EditorPage = observer(() => {
           display: "flex",
           justifyContent: "space-between",
           alignItems: "center",
+          padding: "0 16px",
+          marginBottom: "16px",
         }}
       >
-        <SearchBar searchArgs={searchArgs} setSearchArgs={setSearchArgs} />
-        <Button type="primary" onClick={handleOpenNewMineralSiteForm}>
-          Add Mineral Site
-        </Button>
+        <div style={{ flex: 1 }}>
+          <SearchBar searchArgs={searchArgs} setSearchArgs={setSearchArgs} />
+        </div>
+        <div style={{ display: "flex", alignItems: "center", gap: "16px" }}>
+          <Button type="primary" onClick={handleOpenNewMineralSiteForm}>
+            Add Mineral Site
+          </Button>
+          <ProfileMenu />
```

Comment:
I think the ProfileMenu should not displayed in Add Mineral Site. It should be in the Navbar